### DISCUSSION TITLE
4CHPROR2 Template for shutter

### DISCUSSION THREAD

**Discussion by manny199** [arendst/Tasmota#12477] [Q&A] [answered]
Hello, I need to insert template string into "configure other" to use shutter slides mode.
I find only template for Sonoff DUAL but NOT for my 4CHPRO2 model that have 4 channels.
Can anyone help me?

**Comment by barbudor** [comment, marked as answer]
You can also configure GPIOs in the GUI in Template without a template string.
The question is more : what exactly do you want to configure ?
Setting Shutter modes is mainly about commands in the console, once after you have relays defines

**Reply by barbudor** [reply]
@manny199 
Looks like the specific template for Shutters on Sonoff Dual R2 is different than the official one because it creates additional virtual Buttons. The Sonoff Dual R2 as only 1 button but tasmota needs 2 buttons and 2 relays per shutter.
As the Sonoff 4CH already have 4 buttons and 4 shutters, you don't need a specific template for managing shutters.
Just proceed as per the doc

**Reply by manny199** [reply]
@barbudor Thank you for the quick answer. 
Unfortunately I am not very familiar with the console commands.
My Sonoff 4CHPRO R2 have 4 buttons/relays for 2 shutters.
I want to manage 4 relays, that is: 2 relays for up and down rolling shutter number 1 and 2 relays for up and down rolling shutter number 2. 
Can you help me with the commands to enter?

**Comment by TrDA-hab** [comment]
@manny199

https://github.com/arendst/Tasmota/discussions/12483
